megamek/src/megamek/common/util/DiscordExportUtil.java

```diff
@@ -0,0 +1,32 @@
+package megamek.common.util;
+
```

Comment:
Please add the copyright from 
https://github.com/MegaMek/megamek/wiki/MegaMek-Coding-Style-Guide